### PR TITLE
[4] compare the correct class in instanceof

### PR DESCRIPTION
**Pull request by PhilETaylor**
code review

### CODE REVIEW

File: administrator/components/com_installer/src/Model/InstallerModel.php [hunk 1]
```diff
@@ -15,6 +15,7 @@
 use Joomla\CMS\Language\Text;
 use Joomla\CMS\MVC\Factory\MVCFactoryInterface;
 use Joomla\CMS\MVC\Model\ListModel;
+use Joomla\Component\Installer\Administrator\Service\HTML\Updatesites;
```
Comment (joomdonation) [inline administrator/components/com_installer/src/Model/InstallerModel.php]: This is not right. The code on this line https://github.com/joomla/joomla-cms/blob/4.0-dev/administrator/components/com_installer/src/Model/InstallerModel.php#L94 should be changed to the following code instead:

```php
if ($this instanceof UpdatesitesModel)
```

Comment (PhilETaylor) [reply administrator/components/com_installer/src/Model/InstallerModel.php]: Good spot - thanks. I obviously was too zzz to code at this point.

**Comment by joomdonation** [comment]
I have tested this item :white_check_mark: successfully on e936f7c5166aa233fd7cdde9357b7c1cc3cb6b3a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34012">issues.joomla.org/tracker/joomla-cms/34012</a>.</sub>

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on e936f7c5166aa233fd7cdde9357b7c1cc3cb6b3a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34012">issues.joomla.org/tracker/joomla-cms/34012</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34012">issues.joomla.org/tracker/joomla-cms/34012</a>.</sub>

**Pull request merged**